Application/SolutionScripts/InitializeDevelopmentEnvironment.psm1

```diff
@@ -173,6 +198,35 @@ function Invoke-ConfigTransform {
     }
 }
 
+function Invoke-NewDevelopmentAppSettings([hashtable] $Settings = @{ }) {
```

Comment:
lets add a comment on what file we are generating.

Reply:
added description